ironfish/src/mining/utils.ts

```diff
@@ -5,8 +5,11 @@
 import bufio from 'bufio'
 import { SerializedBlockTemplate } from '../serde/BlockTemplateSerde'
 
+const MINEABLE_BLOCK_HEADER_SIZE = 224
+export const MINEABLE_BLOCK_HEADER_GRAFFITI_OFFSET = MINEABLE_BLOCK_HEADER_SIZE - 32
+
 export function mineableHeaderString(header: SerializedBlockTemplate['header']): Buffer {
-  const bw = bufio.write(240)
+  const bw = bufio.write(MINEABLE_BLOCK_HEADER_SIZE)
```

Comment:
Did we miss changing this in a previous PR and that's why it's changing?

Reply:
Yep, I bet I missed changing it in at least one of mine, if not 2. Same with the graffiti offset, it was 176 but is now 192.